src/drivers/dw/dma.c

```diff
@@ -1289,6 +1289,11 @@ static int dw_dma_avail_data_size(struct dma *dma, unsigned int channel)
 	if (size < 0)
 		size += chan->ptr_data.buffer_bytes;
 
+	if (!size)
+		trace_dwdma("dw_dma_avail_data_size() "
```

Comment:
@keyonjie Have you confirmed that not clearing DONE bit on time actually causes deassertion of DW_DMA_CHAN_EN bit? If it's not then there is no point in logging channel enable status.

Reply:
@tlauda during my test there is some cases that the DMA channel is already [triggered] stopped but there is still calling to this dw_dma_avail_data_size(), that's why I added this logging.

Reply:
@keyonjie OK, so we should add similar log to the dw_dma_free_data_size.

Reply:
> @keyonjie OK, so we should add similar log to the dw_dma_free_data_size.

Sure, let me add it now.